### ISSUE TITLE
t/table.t fails if PDF::API2 is not installed

### ISSUE THREAD

**Issue by eserte** [kamenov/PDF-Table#29]
On some of my cpan smokers I see the following failure:
```
Can't locate PDF/API2.pm in @INC (you may need to install the PDF::API2 module) (@INC contains: /home/cpansand/.cpan/build/2016051900/PDF-Table-0.9.11-kT1lSB/blib/lib /home/cpansand/.cpan/build/2016051900/PDF-Table-0.9.11-kT1lSB/blib/arch /usr/perl5.24.0p/lib/site_perl/5.24.0/amd64-freebsd /usr/perl5.24.0p/lib/site_perl/5.24.0 /usr/perl5.24.0p/lib/5.24.0/amd64-freebsd /usr/perl5.24.0p/lib/5.24.0 .) at t/table.t line 3.
BEGIN failed--compilation aborted at t/table.t line 3.
t/table.t ...... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Comment by kamenov** [comment]
Thanks for reporting that issue - my bad that i didn't follow the best practices and run a make test.

Issues fixed with commit b15ddc8bf466f2e1dc924551a859e5d0dad4aa3d and released to CPAN with Version 0.9.12

**Comment by kamenov** [comment]
The following report has been written by the PAUSE namespace indexer.
Please contact modules@perl.org if there are any open questions.

 User: OMEGA (Desislav Kamenov)
 Distribution file: PDF-Table-0.9.12.tar.gz
 Number of files: 22
 *.pm files: 3
 README: PDF-Table-0.9.12/README
 META-File: PDF-Table-0.9.12/META.json
 META-Parser: Parse::CPAN::Meta 1.4414
 META-driven index: no
 Timestamp of file: Thu May 19 11:11:04 2016 UTC
 Time of this run: Thu May 19 11:12:34 2016 UTC

Status of this distro: OK
